### PR TITLE
Add project registration post projects/ route update

### PR DESCRIPTION
**Pull request by magarnier**

### CODE REVIEW

File: languages/en/deployment-guide/intro.rst [hunk 1]
```diff
@@ -6,6 +6,13 @@ Note about config files (Tuleap's \*.inc): as long as you are OK with the defaul
 the development team, there is no need for you to add those new variables in the corresponding
 file, the default is automatically set for you.
 
+Tuleap 11.9
+===========
+
+Rest route POST /projects have been reworked :
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: has been

File: languages/en/deployment-guide/intro.rst [hunk 2]
```diff
@@ -6,6 +6,13 @@ Note about config files (Tuleap's \*.inc): as long as you are OK with the defaul
...
+ - it does no longer need the delegation permission of REST administrator to be used (but it respect project configuration rule and check if user can create new projects).
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: respects

File: languages/en/user-guide/rest/api.rst [hunk 3]
```diff
@@ -151,3 +151,29 @@ Some rule are specific to this route:
  - limit is set by platform administrators and is common for all projects
  - if plugin archive deleted items is enabled, XML is archived
  
+POST /projects
+--------------
+Here an example of of public created project:
```
Comment (robinsoc) [inline languages/en/user-guide/rest/api.rst]: an "of" is enough

Comment (robinsoc) [reply languages/en/user-guide/rest/api.rst]: I suggest : Here an example of the response when a public project is created 

What do you think ?

Comment (magarnier) [reply languages/en/user-guide/rest/api.rst]: It's an example of usage, it's my payload, not my response

Comment (robinsoc) [reply languages/en/user-guide/rest/api.rst]: Ack

File: languages/en/user-guide/rest/api.rst [hunk 4]
```diff
@@ -151,3 +151,29 @@ Some rule are specific to this route:
  - limit is set by platform administrators and is common for all projects
  - if plugin archive deleted items is enabled, XML is archived
  
+POST /projects
+--------------
+Here an example of the pyload to create a public project:
```
Comment (robinsoc) [inline languages/en/user-guide/rest/api.rst]: payload

**Pull request merged**